examples/usage.ipynb

```diff
@@ -260,6 +251,122 @@
+    "\n",
+    "Construct an `IQMDevice` instance representing the Apollo architecture, print its qubit connectivity and description"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    },
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "apollo = Apollo()\n",
+    "print(apollo.__doc__)\n",
+    "print(apollo.NATIVE_GATES)\n",
+    "print(apollo.NATIVE_GATE_INSTANCES)\n",
+    "print(apollo.qubits)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create a circuit and insert native gates"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a, b, c, d, e, f = apollo.qubits[:6]\n",
+    "circuit = cirq.Circuit()\n",
+    "circuit.append(cirq.CZ(a, b))\n",
+    "circuit.append(cirq.CZ(a, c))\n",
+    "circuit.append(cirq.YPowGate(exponent=1.1)(d))\n",
+    "circuit.append(cirq.YPowGate(exponent=1.1)(e))\n",
+    "circuit.append(cirq.PhasedXPowGate(phase_exponent=0.3, exponent=0.5)(f))\n",
+    "print(circuit)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Insert non-native gates and decompose them into native ones by calling `decompose_circuit`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "circuit.append(cirq.ZZPowGate(exponent=0.2, global_shift=-0.5)(d, e))\n",
+    "circuit.append(cirq.HPowGate(exponent=-0.4)(f))\n",
+    "circuit = apollo.decompose_circuit(circuit)\n",
+    "print(circuit)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Use the `simplify_circuit` method to run a sequence of optimization passes on a circuit"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "circuit = apollo.decompose_circuit(circuit)\n",
+    "circuit = simplify_circuit(circuit)\n",
+    "print(circuit)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "circuit.append(cirq.measure(a, f, key='result'))\n",
+    "\n",
+    "sim = cirq.Simulator()\n",
+    "samples = sim.run(circuit, repetitions=100)\n",
+    "\n",
+    "print('Samples:')\n",
+    "print(samples.histogram(key='result'))\n",
+    "print('\\nState before the measurement:')\n",
+    "result = sim.simulate(circuit[:-1])\n",
+    "print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that the above output vector represents the state before the measurement in the optimized circuit, not the original one, which would have the same phase for both terms. `simplify_circuit` has eliminated a `ZPowGate` which has no effect on the measurement."
```

Comment:
Let's remove this from here. I think the one in the Adonis section is enough. I didn't even check if it is true in this example as well, but the example is more complicated compared to the Adonis example and it is hard to calculate in your head what this comment says.

Reply:
Removed.